### DISCUSSION TITLE
demo installation

### DISCUSSION THREAD

**Discussion by tehluny** [alm0ra/paper_trading_view#4] [Q&A] [answered]
hello could you post a demo video on how to install things properly for those of us not familiar with coding and python usage
the idea of your application is brilliant ive been searching ages for such
pretty sure it is worth some money
only thing i struggle with is to run it on python as i constantly get message of wrong sintaxis when copypasting or im doing it totally wrong

**Comment by tehluny** [comment]
im not familiar with coding in python 
copypasting lines directly into it doesnt work so i have no idea how to actually make your great product run

**Comment by alm0ra** [comment, marked as answer]
hello dear @tehluny , good to see you here
yeah, of course , I will do it after a while but now I made it easy for you . you can just download a `.rar` file extract it and run `ptv.exe` and Boom!, congratulations you made it :))))
checkout this link : 
[quick-installation-for-windows-users](https://github.com/xibalbas/paper_trading_view#quick-installation-for-windows-users)

 Demo video:
[Qucik_installation_guide_video](https://youtu.be/zW962uAD4TU)

if there are other bugs or issues please feel free to open an issue , I will do my best
